core/node/proof_data_handler/src/tee_request_processor.rs

```diff
@@ -78,6 +92,66 @@ impl TeeRequestProcessor {
         result
     }
 
+    #[tracing::instrument(skip(self))]
+    async fn tee_verifier_input_for_existing_batch(
+        &self,
+        l1_batch_number: L1BatchNumber,
+    ) -> Result<TeeVerifierInput, RequestProcessorError> {
+        let vm_run_data: VMRunWitnessInputData = self
+            .blob_store
+            .get(l1_batch_number)
+            .await
+            .map_err(RequestProcessorError::ObjectStore)?;
+
+        let merkle_paths: WitnessInputMerklePaths = self
+            .blob_store
+            .get(l1_batch_number)
+            .await
+            .map_err(RequestProcessorError::ObjectStore)?;
+
+        let mut connection = self
+            .pool
+            .connection()
```

Comment:
Nit: Please use `connection_tagged` for non-test code; it allows more precise debugging / tracing.